src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoColumnCustomNode.cs

```diff
@@ -20,6 +22,81 @@ public override string GetNodeCustomName(object smoObject, SmoQueryContext smoCo
         {
             return SmoColumnCustomNodeHelper.CalculateCustomLabel(smoObject, smoContext);
         }
+
+        public override IEnumerable<NodeSmoProperty> SmoProperties
+        {
+            get
+            {
+                return new List<NodeSmoProperty>
```

Comment:
This will create a new List every time SmoProperties is called. Can we make it either a lazily initialized object or construction is when ColumnsChildFactory is constructed and keep returning that readonly private variable in this property.